doc/user/data/index_usage_key_quantity.yml

```diff
@@ -0,0 +1,66 @@
+# CREATE INDEX idx_orders_view_qty on orders_view (quantity);
+queries:
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view;
+      ```
+    index_usage: Index scan.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE quantity = 10;
+      ```
+    index_usage: Point lookup.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE quantity IN (10, 20);
+      ```
+    index_usage: Point lookup.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE quantity = 10 OR quantity = 20;
+      ```
+    index_usage: |
+      Point lookup. Query uses `OR` to combine conditions on the **same** field.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE quantity = 10 AND price = 5.00;
+      ```
+    index_usage: |
+      Point lookup on `quantity`, then filter on `price`.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE (quantity, price) = (10, 5.00);
+      ```
+    index_usage: |
+      Point lookup on `quantity`, then filter on `price`.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE quantity = 10 OR price = 5.00;
+      ```
+    index_usage: |
+      Index scan. Query uses `OR` to combine conditions on **different** fields.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE quantity <= 10;
+      ```
+    index_usage: Index scan.
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE round(quantity) = 20;
+      ```
+    index_usage: Index scan.
+
+
+  - query: |
+      ```mzsql
+      SELECT * FROM orders_view WHERE quantity = 'hello';
```

Comment:
This might be clearer as `quantity::TEXT = 'hello'` (or maybe just explain that `quantity` is being implicitly cast to text).

Reply:
added in the patch